pkg/cmd/cli/describe/printer.go

```diff
@@ -161,6 +161,13 @@ func PrintTemplateParameters(params []templateapi.Parameter, output io.Writer) e
 	return nil
 }
 
+// Receives a resource kind and its name, returns a formatted
+// string with the "resource_kind/resource_name"
+func formatResourceName(kind, name string) string {
+	rname := kind + "/" + name
```

Comment:
@rhcarvalho Thanks again for the feedback! I do think having something like the above suggestion would be cleaner. I'll go ahead and put it together.